src/tags/app-code-block/app-code-block.style.scss

```
@@ -91,6 +91,10 @@
           opacity: 0;
         }
       }
+      // Leaves the caret solid, which is what the keyframes rest at.
```

Stylelint (17.14.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)